extension/parquet/parquet_extension.cpp

```diff
@@ -1438,6 +1496,27 @@ void ParquetExtension::Load(DuckDB &db) {
 	function.extension = "parquet";
 	ExtensionUtil::RegisterFunction(db_instance, function);
 
+	CopyFunction geo_function("geoparquet");
+	geo_function.plan = WriteGeoParquetPlan;
```

Comment:
This now requires users to pick a separate `geoparquet` format - would it not be better to automatically push `ST_AsWKB` in the parquet writer when writing geometry types? If geometry is being written spatial should be present anyway - and I can imagine that geoparquet is what people want/expect when writing to parquet.

Or do you think there is a reason to keep the old behavior intact for the `PARQUET` format? We could always move the old behavior behind a flag as well (e.g. `GEOMETRY_AS_BLOB`).

Reply:
Yeah good idea, I was trying to avoid having to add any new bind options but I realize that I'm probably going to end up needing to do that anyway in the future so we might as well just use the normal parquet bind and push the conversions there.